ng-test/src/app/address.geo/address-geo.component.ts

```diff
@@ -46,17 +31,19 @@ export class AddressGeoComponent implements OnInit {
           this.address.geo.lng = res.results[0].geometry.location.lng;
           let lenghtTab: number = res.results[0].address_components.length - 1;
           if (this.address.postalCode == res.results[0].address_components[lenghtTab].long_name) {
-            this.addressReady = true;
-             this.addressReadyEvent.emit(this.addressReady);
+             this.onAddressReady.emit(true);
           } else {
+            this.onAddressReady.emit(false);
```

Comment:
pourquoi tu ne passes pas `this.onAddressReady.emit(this.address);` quand c'est ok `this.onAddressReady.emit();` et vide ou false quand il y a n problème ?